check_stability.py

```diff
@@ -17,6 +18,8 @@
 
 import requests
 
+_CONFIG_FILE = "check_stability.ini"
```

Comment:
Instead of making this a global, I might be tempted to make it a command line option with a default value.